src/InteractorDaemon.js

```diff
@@ -387,7 +401,7 @@ const InteractorDaemon = module.exports = class InteractorDaemon {
         setTimeout(cb, 20)
       }
     }
-    return this._verifyEndpoint(verifyEndpointCallback)
+    return this._verifyEndpoint(true, verifyEndpointCallback)
```

Comment:
If a user re-link is server, it will not be the first connection for this server, use `agent.json5` would be better I think